### ISSUE TITLE
_index.html rendering is broken with Hugo 0.60.0 or later

### ISSUE THREAD

**Issue by tekezo** [google/docsy#157]
Markdown library has been changed to Goldmark from Blackfriday at Hugo 0.60.0.
`_index.html` rendering is broken due to the library change.

For example, docsy userguide with Hugo 0.60.1:

<img width="1539" alt="Screen Shot 2019-12-04 at 0 54 50" src="https://user-images.githubusercontent.com/659178/70067114-24831380-1631-11ea-8876-170baf96769d.png">

## Workaround

Put the following lines into config.toml.

```
[markup]
  [markup.goldmark]
    [markup.goldmark.renderer]
      unsafe = true
```

Then the rendering is partially fixed as follows, but download link on docsy userguide is still broken.

<img width="1479" alt="Screen Shot 2019-12-04 at 0 55 06" src="https://user-images.githubusercontent.com/659178/70067123-277e0400-1631-11ea-87e1-2a9c7170e4a4.png">

**Comment by LisaFC** [comment]
Aaaaaagggghhh, Markdown inconsistency, my enemy!!!!

Thanks for spotting that, I'll have a look and see if I can fix it so it works with Blackfriday AND Goldmark.

**Comment by LisaFC** [comment]
Have added the config.toml workaround so that at least Goldmark renders raw HTML, will now look at the broken link...

**Comment by LisaFC** [comment]
So entertainingly the problem seemed to be a blank line. Ah, Markdown. I'll just check that it still works with 0.55....

**Comment by LisaFC** [comment]
Fixed in #170
